lib/chef/data_collector.rb

```diff
@@ -210,10 +209,10 @@ def send_to_data_collector(message)
       # @param message [Hash] message to send
       #
       def send_to_output_locations(message)
-        return unless Chef::Config[:data_collector][:output_locations]
+        Chef::DataCollector::ConfigValidation.validate_output_locations!
 
         Chef::Config[:data_collector][:output_locations].each do |type, locations|
-          locations.each do |location|
+          Array(locations).each do |location|
```

Comment:
Allow locations as a single value String or Array for more flexible options value.